modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -469,6 +469,27 @@ The `Iterator` type is based on the https://docs.oracle.com/javase/8/docs/api/ja
 
 Note that like the Java class, the iterator is designed to be consumed only once. For example, if you then pass this value to a logger would result in consuming it and it would no longer be readable to further elements in the flow.
 
+[[dw_type_null]]
+== Null (dw::Core Type)
+
+DataWeave defines a `Null` data type for `null` values. It is important to note that `null` 
+is the value, while `Null` is the type. DataWeave functions do not recognize `Null` as
+a `null` value. 
+
+You can determine whether a DataWeave function can handle `null` values by
```

Comment:
I know that we refer to various kinds of "handlers," but I still think that "handle" as a verb might be more difficult to translate (by audiences) than something like "process" or "manage" or similar more context-specific term. Just a suggestion for your consideration.

Reply:
Maybe "accept". "handle" is very commonly used term in programming, and a translator should be able to translate.